data/forms/medical-needs.json

```diff
@@ -21,6 +21,21 @@
           "validation": {
             "required": true
           }
+        },
+        {
+          "name": "proof-of-medical-needs",
+          "as": "insettext",
```

Comment:
In the designs this looks to be using announcement styles, but this should be ok.

Reply:
All of the other announcements in the site use the same inset text component. There wasn't a way in the design system to apply the green styling